external/libtorch_cpu.BUILD

```diff
@@ -4,13 +4,15 @@ licenses(["notice"])
 
 cc_library(
     name = "libtorch_cpu",
+    srcs = glob(["lib/*.so"]),
+    hdrs = glob(["*"]),
     includes = [
-        ".",
-        "torch/csrc/api/include",
+        "include",
+        "include/torch/csrc/api/include",
+        "include/torch/csrc/api/include/torch",
```

Comment:
include directories sharing  same ancestor  should be as few as possible. Flexibility is a bad idea here.